src/main/java/uk/gov/hmcts/reform/feature/config/FlywayConfiguration.java

```diff
@@ -24,7 +24,14 @@
 public class FlywayConfiguration {
 
     @Bean
+    @ConditionalOnProperty(prefix = "flyway", name = "noop.strategy", matchIfMissing = true)
     public FlywayMigrationStrategy flywayMigrationStrategy() {
         return new FlywayNoOpStrategy();
     }
+
+    @Bean
+    @ConditionalOnProperty(prefix = "flyway", name = "noop.strategy", havingValue = "false")
+    public FlywayMigrationStrategy flywayVoidMigrationStrategy() {
```

Comment:
will this cause an actual migration to happen?
@lgonczar Did this differently in the draft-store:
https://github.com/hmcts/draft-store/commit/bd3e129960d4ead43c53330d71b64014ba64f823